ludwig/utils/image_utils.py

```diff
@@ -154,4 +159,51 @@ def num_channels_in_image(img):
     if img.ndim == 2:
         return 1
     else:
-        return img.shape[2]
+        return img.shape[0]
+
+
+def img_prop_to_numpy(prop: Union[int, Tuple[int]]) -> np.ndarray:
+    """ Creates a np array of length 2 from a Conv2D property.
+
+    E.g., stride=(2, 3) gets converted into np.array([2, 3]), where the
+    height_stride = 2 and width_stride = 3.
+    """
+    if type(prop) == int:
+        return np.ones(2) * prop
+    elif type(prop) == tuple and len(prop) == 2:
+        return np.array(list(prop))
+    else:
+        raise TypeError(f'kernel_size must be int or tuple of length 2.')
+
+
+def get_img_output_shape(
```

Comment:
I'm wondering if we can do something similar to what `pytorch-summary` provides here for more complicated modules such as CNNs:

https://github.com/sksq96/pytorch-summary

I will post about this on the Ludwig channel, please share your thoughts as well!

Reply:
Solved on slack.